scripts/install-worker.sh

```diff
@@ -25,7 +25,8 @@ set -x
 # alternative (no releases and no tags).
 mkdir /backend
 cd /backend
-git clone --depth 1 https://github.com/lsst-dm/image_cutout_backend.git
+# TODO: This should not reference a ticket branch..
+git clone --depth 1 -b tickets/DM-42191 https://github.com/lsst-dm/image_cutout_backend.git
```

Comment:
I'll add a tag in image-cutout-backend that we can reference after the changes there get through review.